build.csx

```diff
@@ -0,0 +1,225 @@
+using System.Xml;
+using System.Xml.Linq;
+using static SimpleTargets;
+
+static var solutionName = "FakeItEasy";
+
+static var solution = "./" + solutionName + ".sln";
+static var packagesDir = Path.GetFullPath("packages");
+var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+var assemblyInfo = "./src/CommonAssemblyInfo.cs";
+var nuspecs = Directory.GetFiles("./src", "*.nuspec");
+var gitlinks = Directory.GetDirectories("./src").Where(p => !p.EndsWith(".Shared")).Select(Path.GetFileName);
+
+var unitTests = new []
+{
+  "tests/FakeItEasy.Tests/bin/Release/FakeItEasy.Tests.dll",
+  "tests/FakeItEasy.Analyzer.CSharp.Tests/bin/Release/FakeItEasy.Analyzer.CSharp.Tests.dll",
+  "tests/FakeItEasy.Analyzer.VisualBasic.Tests/bin/Release/FakeItEasy.Analyzer.VisualBasic.Tests.dll"
+};
+
+var netstdUnitTestDirectories = new []
+{
+  "tests/FakeItEasy.Tests.netstd"
+};
+
+var integrationTests = new []
+{
+  "tests/FakeItEasy.IntegrationTests/bin/Release/FakeItEasy.IntegrationTests.dll",
+  "tests/FakeItEasy.IntegrationTests.VB/bin/Release/FakeItEasy.IntegrationTests.VB.dll"
+};
+
+var netstdIntegrationTestDirectories = new []
+{
+  "tests/FakeItEasy.IntegrationTests.netstd"
+};
+
+var specs = new []
+{
+  "tests/FakeItEasy.Specs/bin/Release/FakeItEasy.Specs.dll"
+};
+
+var netstdSpecDirectories = new []
+{
+  "tests/FakeItEasy.Specs.netstd"
+};
+
+var approvalTests = new []
+{
+  "tests/FakeItEasy.Tests.Approval/bin/Release/FakeItEasy.Tests.Approval.dll",
+  "tests/FakeItEasy.Tests.Approval.netstd/bin/Release/FakeItEasy.Tests.Approval.netstd.dll"
+};
+
+// tool locations
+static var msBuild = $"{Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86)}/MSBuild/14.0/Bin/msbuild.exe";
+var nuget = "./.nuget/NuGet.exe";
+var gitlink = "./packages/gitlink.2.3.0/lib/net45/GitLink.exe";
+static var xunit = "./packages/xunit.runner.console.2.0.0/tools/xunit.console.exe";
+
+// artifact locations
+var logsDirectory = "./artifacts/logs";
+var outputDirectory = "./artifacts/output";
+static var testsDirectory = "./artifacts/tests";
+
+// targets
+var targets = new TargetDictionary();
+
+targets.Add("default", DependsOn("unit", "integ", "spec", "approve", "pack", "gitlink"));
+
+targets.Add("outputDirectory", () => Directory.CreateDirectory(outputDirectory));
+
+targets.Add("logsDirectory", () => Directory.CreateDirectory(logsDirectory));
+
+targets.Add("testsDirectory", () => Directory.CreateDirectory(testsDirectory));
+
+targets.Add("build", DependsOn("clean", "outputDirectory", "restore"), () => RunMsBuild("Build"));
+
+targets.Add("clean", DependsOn("logsDirectory"), () => RunMsBuild("Clean"));
+
+targets.Add(
+    "restore",
+    () =>
+    {
+        Cmd(nuget, $"restore {solution} -PackagesDirectory {packagesDir}");
+        foreach (var projectDir in netstdUnitTestDirectories.Concat(netstdIntegrationTestDirectories).Concat(netstdSpecDirectories))
+        {
+            Cmd("dotnet", $"restore {projectDir}");

```

Comment:
Lemme tell you about that. I don't know.
Also, this command is not needed. I just verified about 15 minutes ago. I was going to submit another PR, as I didn't want to change what we did here too much (just how), but I can change this PR to omit this `foreach`, if it suits.